### DISCUSSION TITLE
What's the highest number of static pages you've pre-rendered in build step?

### DISCUSSION THREAD

**Discussion by tomgreeEn** [vercel/next.js#21751] [Help]
Hi

I'm interested in many pages it's possible to pre-render during the build step in the real world. What's the highest number you've used/are using? 

I'm using Vercel which gives 45 minutes maximum build time, which I think is the longest for any managed solution at the moment, so I guess ultimately it is a factor of how many page prop requests your DB can fulfil during this time? 

Thanks

Tom

**Comment by sanderkooger** [comment]
At the moment, we are pre-rendering everything. For thisisfashion.tv But I see us running onto issues fairly quickly. Later a strategy where shows aired in the last year might be a better solution with a fallback for older paths
